lambdas/tests/unit/handlers/test_search_patient_details_handler.py

```diff
@@ -18,126 +17,77 @@ def patch_env_vars():
     with patch.dict(os.environ, env_vars):
         yield env_vars
 
+@pytest.fixture()
+def mocked_context(mocker):
+    mocked_context = mocker.MagicMock()
+    mocked_context.authorization = {
+        "selected_organisation": {"org_ods_code": "Y12345"},
+        "repository_role": "GP_ADMIN",
+    }
+    _ = mocker.patch(
+        "handlers.search_patient_details_handler.request_context", mocked_context
+    )
+    yield _
```

Comment:
I think here we could yield the patched object directly

```suggestion
    yield mocker.patch(
        "handlers.search_patient_details_handler.request_context", mocked_context
    )
```